core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerClientPool.java

```diff
@@ -48,12 +48,14 @@
    *
    * @param userState the parent userState
    * @param address address of the worker
+   * @param minCapacity the minimum capacity of the pool
    * @param maxCapacity the maximum capacity of the pool
    * @param alluxioConf Alluxio configuration
    */
-  public BlockWorkerClientPool(UserState userState, GrpcServerAddress address, int maxCapacity,
-      AlluxioConfiguration alluxioConf) {
-    super(Options.defaultOptions().setMaxCapacity(maxCapacity).setGcExecutor(GC_EXECUTOR));
+  public BlockWorkerClientPool(UserState userState, GrpcServerAddress address, int minCapacity,
+      int maxCapacity, AlluxioConfiguration alluxioConf) {
```

Comment:
Can't we just set minCapacity to 0 here rathat than defining a new prop?
I don't see a scenario where we'd set it anything else.

Reply:
@ggezer I updated the default to zero as like you said that is the common scenario and made it hidden. just to be consistent with other client thread pools, i didn't remove the property